stanford/bin/dump-logs

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+rsync_sudo() {
+    rsync --rsync-path='sudo rsync' -avh --progress ${@}
```

Comment:
You need the `sudo` bit, else you run into permissions accessing certain log files.
The rest is sugar.